notebooks/official/stable_diffusion/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<p>890M parameters is not realistic to run on commodity machine and GPU.</p><p>Consider at least a n1-standard-8 and a P100</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>Done.</p>